.github/issue_template/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+
+about: File a bug report.
+
+---
+
+ <!-- If you suspect this could be a bug, follow the template. -->
+
+### What version of Dgraph are you using?
+
+
+### Tell us a little more about your go-environment?
```

Comment:
Maybe add "execute this: `go version`"

Reply:
+1 ^ addressing it here https://github.com/dgraph-io/dgraph/pull/8159/commits/7be517f1a5463b99982fc3252de73a0b2d299984